linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_etc_gshadow_open/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: rhel7,rhel8,fedora,ol7,ol8
+
+title: 'Record Events that Modify User/Group Information via open syscall - /etc/gshadow'
+
+description: |-
+    The audit system should collect write events to /etc/gshadow file for all users and root.
+    If the <tt>auditd</tt> daemon is configured
+    to use the <tt>augenrules</tt> program to read audit rules during daemon
+    startup (the default), add the following lines to a file with suffix
+    <tt>.rules</tt> in the directory <tt>/etc/audit/rules.d</tt>:
+    <pre>-a always,exit -F arch=b32 -S open -F a1&amp;03 -F path=/etc/gshadow -F auid>={{{ auid }}} -F auid!=unset -F key=user-modify</pre>
```

Comment:
Shouldn't there be both 32 and 64 bit?

Reply:
The 64bit audit rule is described right after, at line 17.

Reply:
Yes, I see, but this one is for `augenrules` and on line 17 it's for `auditctl`. I thought that both rules should be set for both tools.

Reply:
I see now, well noted!

Reply:
@jan-cerny Should be fixed. I have also piggy backed same description fix for rules for `/etc/group` and `/etc/passwd`.